.gitlab-ci.yml

```diff
@@ -77,29 +77,23 @@ variables:
           --tag "$IMAGE_NAME:latest"
           --file "$DOCKERFILE" .
 
-check-typing:
-  stage:                           test
-  <<:                              *common-refs
-  <<:                              *kubernetes-env
-  script:
-    - yarn --immutable --network-concurrency 1
-    - yarn typing
-
 check-linting:
   stage:                           test
   <<:                              *common-refs
   <<:                              *kubernetes-env
   script:
-    - yarn --immutable --network-concurrency 1
-    - yarn lint
+    - apk add --no-cache python3 openssl make g++
+    - yarn --immutable
+    - yarn typing && yarn format && yarn lint
 
-check-formatting:
+check-tests:
   stage:                           test
   <<:                              *common-refs
   <<:                              *kubernetes-env
   script:
-    - yarn --immutable --network-concurrency 1
-    - yarn format
+    - apk add --no-cache python3 openssl make g++
```

Comment:
Add a `TODO` for creating an image which already has those packages and create a ticket for it

Reply:
[Added](https://github.com/paritytech/substrate-matrix-faucet/pull/150/files#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7R85)